python/lsst/ctrl/mpexec/cli/pipetask.py

```diff
@@ -0,0 +1,40 @@
+# This file is part of ctrl_mpexec.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import click
+
+from lsst.daf.butler.cli.butler import LoaderCLI
+from lsst.daf.butler.cli.opt import log_level_option
+
+
+class PipetaskCLI(LoaderCLI):
+
+    localCmdPkg = "lsst.ctrl.mpexec.cli.cmd"
+
+
+@click.command(cls=PipetaskCLI, context_settings=dict(help_option_names=["-h", "--help"]))
+@log_level_option()
+def cli(log_level):
+    PipetaskCLI.initLogging(log_level)
```

Comment:
Part of the switch here it to unify interfaces for command line options that are identical. We still have the opportunity to change this interface.   I think I prefer `--log-level` and I'm fine with `-L`. If this implementation has more features I'm fine with `butler` subcommands getting the new features. Note that this will require that the daf_butler version does the `lsst.log` configuration as well (but with a protected import since `lsst.log` is not to be required by daf_butler).